puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging_build_rpm.yaml

```diff
@@ -1,15 +1,7 @@
-# Builds a single Foreman (sub)project or dependency from the foreman-packaging
-# repository, using Koji.
-#
-# See https://projects.theforeman.org/projects/foreman/wiki/RPM_Packaging#Project-sources-version-3
-# or the help text on the build page for usage.
 - job:
     name: packaging_build_rpm
-    concurrent: true
```

Comment:
You probably want to keep this given our recent learning.

Reply:
urite